### ISSUE TITLE
Hyphens should not be reset on default text-align

### ISSUE THREAD

**Issue by JayPanoz** [edrlab/thorium-web#151]
Right now hyphens state is bound to text align, and is being reset when switching back to default – where it is disabled. 

The state should be kept if set, either by the user or the side effect of applying the justify option from a previous state of publisher. 

Only the preference that is submitted to the navigator should be nullified, as the publisher styles component does for instance.
